partials/mobile-nav.hbs

```diff
@@ -0,0 +1,116 @@
+<ul class="main_menu">
+    <li id="what_is_holochain_item" class="menu_item"><button>WHAT IS HOLOCHAIN</button>
+        <ul class="sub-menu">
+            <li>
+                <a href="/what-holochain"><span>What is Holochain?</span> <span>Framework and P2P networking</span></a>
```

Comment:
```suggestion
                <a href="https://holochain.org/what-holochain" target="blank"  rel="noopener noreferrer"><span>What is Holochain?</span> <span>Framework and P2P networking</span></a>
```

I see these links are still relative to the blog, which doesn't have any content at those URLs. This one and the ones below it need to point to `https://holochain.org/{url}`.

They should probably all also have `target="_blank"` as this suggestion shows, because they're pointing to another site. I'm willing to be convinced otherwise though :)

oh, and I see that other external links in this menu also have `rel="noopener noreferrer"`. I'm not entirely sure what they do, but I'll trust the wisdom of whoever put them in there and recommend they go on this one too.